source/utils/adios_iotest/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ target_link_libraries(adios_iotest adios2::cxx11_mpi MPI::MPI_C)
 if(WIN32)
   target_link_libraries(adios_iotest getopt)
 endif()
+set_property(TARGET adios_iotest PROPERTY OUTPUT_NAME adios2${ADIOS2_LIBRARY_SUFFIX}_iotest)
```

Comment:
@pnorbert Same as above on renaming `adios_iotest` to `adios2_iotest`

Reply:
I was one time trying to just name everything adios, which is clearly not the case. I guess adios2 used everywhere and everything would be more consistent. Same with _ vs - in the names.

Reply:
We should just be sure to explicitly mention in the release notes the tool renaming as it could be considered an API breakage.